python/hawkey/query-py.c

```diff
@@ -647,6 +650,30 @@ q_difference(PyObject *self, PyObject *other)
 }
 
 static PyObject *
+filter_unneeded(PyObject *self, PyObject *args, PyObject *kwds)
+{
+    HyQuery self_query_copy = hy_query_clone(((_QueryObject *) self)->query);
+    const char *kwlist[] = {"swdb", "debug_solver", NULL};
+    PyObject *swdb;
+    int debug_solver = 0;
+
+    if (!PyArg_ParseTupleAndKeywords(args, kwds, "O|i", (char**) kwlist, &swdb, &debug_solver)) {
```

Comment:
I see in original Python code that debug_solver is used as bool:
`def _unneeded(self, sack, history, debug_solver=False):`
But it is integer now.
Why there you do not use `PyArg_ParseTupleAndKeywords(args, kwds, "O|O!", (char**) kwlist, &swdb, &PyBool_Type, &debug_solver)`?
And debug_solver declared as `PyObject` and `PyObject_IsTrue(debug_solver)` on line 665.
Is it intended change from bool to int?